src/graph/graphcontext.c

```diff
@@ -82,8 +83,17 @@ static GraphContext *_GraphContext_Create(RedisModuleCtx *ctx, const char *graph
 	return gc;
 }
 
+static bool _GraphContext_IsModuleReplicating() {
+	return aux_field_counter > 0;
+}
+
 GraphContext *GraphContext_Retrieve(RedisModuleCtx *ctx, RedisModuleString *graphID, bool readOnly,
 									bool shouldCreate) {
+	if(_GraphContext_IsModuleReplicating()) {
+		// The whole module is currently replicating, emit an error.
+		RedisModule_ReplyWithError(ctx, "ERR RedisGraph module is currently replicating");
```

Comment:
Does this change allow us to delete any of the logic that had been added to support mid-replication queries?

Reply:
yes
I removed this logic